server/app/views/applicant/ProgramIndexView.java

```diff
@@ -603,7 +603,7 @@ private FormTag renderCategoryFilterChips(
                         relevantCategories,
                         category ->
                             div()
-                                .withId("filter-chip")
+                                .withId("filter-chip-" + category)
```

Comment:
(question, blocking) Can the categories have spaces or non-alphanumeric characters? If yes, they should be replaced with `-` or `_`.

Reply:
Right now, all ten categories are a single word, but I can see the value of future-proofing this.  Will add the transformation logic.

Reply:
Done